core/chains/evm/gas/suggested_price_estimator.go

```diff
@@ -113,28 +113,28 @@ func (o *l2SuggestedPriceEstimator) refreshPrice() (t *time.Timer) {
 	}
 	bi := (*assets.Wei)(&res)
 
-	o.logger.Debugw("refreshPrice", "l2GasPrice", bi)
+	o.logger.Debugw("refreshPrice", "GasPrice", bi)
 
 	o.gasPriceMu.Lock()
 	defer o.gasPriceMu.Unlock()
-	o.l2GasPrice = bi
+	o.GasPrice = bi
 	return
 }
 
-func (o *l2SuggestedPriceEstimator) OnNewLongestChain(context.Context, *evmtypes.Head) {}
+func (o *SuggestedPriceEstimator) OnNewLongestChain(context.Context, *evmtypes.Head) {}
 
-func (*l2SuggestedPriceEstimator) GetDynamicFee(_ context.Context, _ uint32, _ *assets.Wei) (fee DynamicFee, chainSpecificGasLimit uint32, err error) {
+func (*SuggestedPriceEstimator) GetDynamicFee(_ context.Context, _ uint32, _ *assets.Wei) (fee DynamicFee, chainSpecificGasLimit uint32, err error) {
 	err = errors.New("dynamic fees are not implemented for this layer 2")
```

Comment:
Almost slipped under my radar since it didn't say "L2"
```suggestion
	err = errors.New("dynamic fees are not implemented for this chain")
```